apps/dashboard/src/components/inbox-button.tsx

```diff
@@ -52,7 +52,7 @@ const InboxInner = () => {
                 <>
                   Inbox
                   {isWorkflowEditorPage && ' (Test)'}
-                  {unreadCount > 0 && ` (${unreadCount})`}
+                  {unreadCount.total > 0 && ` (${unreadCount.total})`}
```

Comment:
the new `renderBell` prop interface provides the `unreadCount` as the object with total and severity information